--- cases/repo_e798854_52c9d19/merge_handler.py	2026-02-28 22:21:31
+++ cases/repo_e798854_52c9d19/llm_handler.py	2026-02-28 22:24:26
@@ -58,7 +58,7 @@
 
     @property
     def instance_id(self):
-        return global_config_map["instance_id"].value or ""
+        return global_config_map["client_id"].value or ""
 
     def emit(self, record):
         if record.__dict__.get("do_not_send", False):
@@ -81,9 +81,6 @@
         """
         sio = io.StringIO()
         tb = ei[2]
-        # See issues #9427, #1553375 in python logging. Commented out for now.
-        # if getattr(self, 'fullstack', False):
-        #     traceback.print_stack(tb.tb_frame.f_back, file=sio)
         traceback.print_exception(ei[0], ei[1], tb, None, sio)
         s = sio.getvalue()
         sio.close()
@@ -136,7 +133,7 @@
                     'Content-Type': "application/json"
                 },
                 "data": json.dumps(logs, default=log_encoder),
-                "params": {"ddtags": f"instance_id:{self.instance_id},"
+                "params": {"ddtags": f"client_id:{self.instance_id},"
                                      f"client_version:{CLIENT_VERSION},"
                                      f"type:log",
                            "ddsource": "hummingbot-client"}
@@ -153,7 +150,7 @@
                     'Content-Type': "application/json"
                 },
                 "data": json.dumps(logs, default=log_encoder),
-                "params": {"ddtags": f"instance_id:{self.instance_id},"
+                "params": {"ddtags": f"client_id:{self.instance_id},"
                                      f"client_version:{CLIENT_VERSION},"
                                      f"type:log",
                            "ddsource": "hummingbot-client"}
@@ -169,7 +166,7 @@
                 "headers": {
                     'Content-Type': "application/json"
                 },
-                "data": json.dumps({"instance_id": self.instance_id,
+                "data": json.dumps({"client_id": self.instance_id,
                                     "exchange": exchange,
                                     "market": market,
                                     "version": CLIENT_VERSION,
@@ -189,7 +186,6 @@
             if global_config_map["send_error_logs"].value:
                 if len(self._log_queue) > 0 and len(self._log_queue) >= min_send_capacity:
                     self.send_logs(self._log_queue)
-
                     self._log_queue = []
             if len(self._event_queue) > 0 and len(self._event_queue) >= min_send_capacity:
                 self.send_events(self._event_queue)
@@ -240,4 +236,4 @@
                 raise
             except Exception:
                 self.logger().network("Unexpected error while sending aggregated metrics.", exc_info=True)
-                return
+                return
\ No newline at end of file
